app/controllers/course/forum/component_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Course::Forum::ComponentController < Course::Forum::Controller
+  before_action :add_forum_name_breadcrumb
+
+  protected
+
+  def add_forum_name_breadcrumb
```

Comment:
One small issue, to be consistent with other components,  how about just `add_forum_breadcrumb` ?